adguard/DOCS.md

```diff
@@ -16,7 +16,8 @@ comparison to installing any other Home Assistant add-on.
 1. **Ensure your Home Assistant device has a
    [static IP and static external DNS servers!](https://github.com/home-assistant/hassos/blob/dev/Documentation/network.md#static-ip)**
    This is important! You **WILL** end up having issues if you skip this step.
-   Also, please note, setting a fixed IP in your router is **NOT** static.
+   - For Home Assistant OS go to: _Supervisor → Host → IP Address → Change → IPv4 → Static_ to access these settings.
```

Comment:
Let's add a link or button to the supervisor info page using a My Home Assistant Link

<https://my.home-assistant.io/create-link/>

Reply:
Now with a button. Noted some [naming inconsistency](https://github.com/home-assistant/my.home-assistant.io/issues/97) btw.